tests/target/legacy-control-style-wrap.rs

```diff
@@ -0,0 +1,12 @@
+// rustfmt-max_width: 100
+// rustfmt-control_style: Legacy
+// Ensure legacy control style produces correct wrapping
+
+fn main() {
+    let aaaaaaaaaa_bbbbbb_cccccc_ddddd = if aaaaaaa_bbbb.cc_dddd() &&
```

Comment:
I believe this isn't true for the legacy control style, is that not the case?

https://github.com/rust-lang-nursery/rustfmt/blob/master/Configurations.md#control_style